joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/datastructures/CSharpGlobal.scala

```diff
@@ -0,0 +1,8 @@
+package io.joern.csharpsrc2cpg.datastructures
+
+import java.util.concurrent.ConcurrentHashMap
+
+class CSharpGlobal {
+  // Mapping method fullname to its return type and signature
+  val methodFullNameReturnTypeMap: ConcurrentHashMap[String, (String, String)] = new ConcurrentHashMap()
```

Comment:
Please see the `TypeMap` as that is what it's for. Then fetch the method you want to look-up via the `scope` object which manages which imports and namespaces are in scope.

Reply:
You may need to modify the `CSharpMethod` to store return information, and add the relevant entry to `resources/builtin_types.json`